README.md

```diff
@@ -19,7 +19,10 @@ Installing all dependencies
 
 - Navigate to `localhost:3456` in Google Chrome browser
 
-## Test
+## Setup Testframework
+- Step by step instructions to setup JEST is found at https://github.com/DynamoDS/librarie.js/wiki/Setting-up-test-frameworks-for-librarie.js
```

Comment:
Few points, it should be *test framework* and not *testframework*. Another way to make it more readable is to hide away full-length URLs, for example:

*Step by step instructions to author tests can be found on [this wiki page](https://github.com/DynamoDS/librarie.js/wiki/Setting-up-test-frameworks-for-librarie.js).*